dev/manual_tests/overlay_geometry.dart

```diff
@@ -163,8 +163,9 @@ class OverlayGeometryAppState extends State<OverlayGeometryApp> {
     });
   }
 
-  void handleTapUp(GlobalKey target, Point globalPosition) {
+  void handleTapUp(Key t, Point globalPosition) {
```

Comment:
I mean the typedef. It's in the same file and only used for this one purpose, so we can make it match.